server/sonar-web/src/main/js/apps/projects/query.ts

```diff
@@ -201,11 +197,11 @@ function convertDuplications(metric: string, duplications: number): string {
     case 1:
       return metric + ' < 3';
     case 2:
-      return metric + ' >= 3';
+      return `${metric} >= 3 AND ${metric} < 5`;
     case 3:
-      return metric + ' >= 5';
+      return `${metric} >= 5 AND ${metric} < 10`;
     case 4:
-      return metric + ' >= 10';
+      return `${metric} >= 10 AND ${metric} < 20`;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Use a template literal instead of concatenating with a string literal. ([typescript:S3512](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=typescript%3AS3512))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarqube%3Asonarqube&pullRequest=3141&issues=AWIKRrzD54IFfP4XjNB1&open=AWIKRrzD54IFfP4XjNB1)

[ISSUE_KEY]: AWIKRrzD54IFfP4XjNB1